docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3"
 services:
   postgres:
-    image: postgres:15.2
+    image: postgres:latest
```

Comment:
I suggest using fixed version of the database, to avoid unexpected breaking changes